### DISCUSSION TITLE
Смена name уже созданной таблицы

### DISCUSSION THREAD

**Discussion by qye1** [totumonline/totum-ru-issues-and-discussions#347] [Как это сделать?] [answered]
Как можно переименовать name уже существующей таблицы?

**Comment by erpic-ru** [comment, marked as answer]
Я знаю только один способ: продублировать таблицу с нужным name и удалить старую

**Reply by qye1** [reply]
Подскажите пожалуйста как продублировать всю таблицу сразу, никак не могу этого сделать.

**Reply by erpic-ru** [reply]
https://ru.docs.totum.online/system-tables#stables-tables

**Reply by qye1** [reply]
Спасибо

**Reply by ttm-alexeypolunin** [reply]
Для Расчетный в цикле для дублирования надо кликнуть на name таблицы в Списке таблиц
